src/script/interpreter.h

```diff
@@ -45,7 +45,8 @@ enum
     SCRIPT_VERIFY_STRICTENC = (1U << 1),
 
     // Passing a non-strict-DER signature to a checksig operation causes script failure (softfork safe, BIP62 rule 1)
-    SCRIPT_VERIFY_DERSIG    = (1U << 2),
+    // In Zcash this is required, and validation of non-strict-DER signatures is not implemented.
```

Comment:
@radix42 what is your decision here, do we want to do anything different than what Zcash is doing with DER signatures? They seem to have more strict standards than BTC, which is a good thing, but you have final say